README.md

```diff
@@ -1 +1,14 @@
 # tap-fitter
+
+RH Tap Fitter is a utility designed to aid operator developers to enable features and generate configurations in order to properly run in cerrtain pipeline environments. 
```

Comment:
```suggestion
RH Tap Fitter is a utility designed to aid operator developers to enable features and generate configurations in order to properly run in certain pipeline environments. 
```

Reply:
oops this is what i get for writing non-code in a text editor and not a word processor